src/htm/algorithms/TemporalMemory.hpp

```diff
@@ -236,6 +236,14 @@ class TemporalMemory : public Serializable
     activateDendrites(learn, externalPredictiveInputsActive, externalPredictiveInputsWinners);
   }
 
+  /**
+	* Calculate anomaly score depending on the mode given at initialization
+	*
+	* @param activeColumns
+	* currently activeColumns
+	*/
+  void calculateAnomalyScore(const SDR &activeColumns);
```

Comment:
- [ ] if possible, make the method `const`
- [ ] upon discussion, but I'd suggest making it `private:` in cpp (and name_ )

Reply:
by private you mean static, right?
It cannot be const since it modifies inner "anomalyScore" variable